src/components/MovieModal.js

```diff
@@ -1,17 +1,20 @@
 import React, { Component } from 'react';
 var $ = require('jquery');
 
+import {apiKey} from '../apiKey';
+
+
 export default class MovieModal extends Component {
 	state = {
 		movie: {}
 	}
 	componentDidMount() {
-		$.get('http://www.omdbapi.com/?i=' + this.props.movieId, (result) => {
+		$.get('http://www.omdbapi.com/?apikey=d73910fd&i=' + this.props.movieId, (result) => {
```

Comment:
Do you notice something here that is bad? :D

Reply:
whoopsie daisy, put a gooD KID